src/main/java/com/sendgrid/SendGridAPI.java

```diff
@@ -0,0 +1,92 @@
+package com.sendgrid;
+
+import java.io.IOException;
+import java.util.Map;
+
+public interface SendGridAPI {
+
+  /**
+   * Initializes SendGrid
+   * 
+   * @param apiKey is your SendGrid API Key: https://app.sendgrid.com/settings/api_keys
+   */
+  public void initializeSendGrid(String apiKey);
+
+  /**
+   * Initializes SendGrid
```

Comment:
This method returns the library version. Please update the documentation to reflect this.